sumo_mgmt.py

```diff
@@ -519,11 +619,42 @@ def print_collector_table(collectors, headings):
   table = AsciiTable(table_data)
   log('[INFO] %d total collectors\n%s' % (table_rows, table.table))
 
+def print_sources_table(collectors, headings):
+  '''
+  Given a set of headings and a list of Collectors, prints the attributes
+  of each Collector (one per row) specified in the headings in a nice
+  ASCII table. If an attribute is empty, a dash is printed in the column 
+  instead.
+
+  Args:
+    collectors (list): The list of Collectors to be printed.
+    headings (list): The list of headings for the table.
+  '''
+  table_data = []
+  table_data.append(headings)
+  table_rows = 0
+
+  if collectors:
+    for collector in collectors:
+      for source in collector['sourcejson']:
+        row = []
+        row.append(collector['id'])
+        for col in headings[1:]:
+          row.append(source[col])
```

Comment:
category may be null.  When I ran `-getsources` with a collector that had null source category I got an error:
```
2019-03-09 17:28:48,647 +1600 [PROGRESS] Finished source get for collector ID: 148919877
Traceback (most recent call last):
  File "sumo_mgmt.py", line 692, in <module>
    print_sources_table(collectors_with_sources, source_table_headings)
  File "sumo_mgmt.py", line 643, in print_sources_table
    row.append(source[col])
KeyError: 'category'
```
Note how we handle this in `print_collector_table` L508-509